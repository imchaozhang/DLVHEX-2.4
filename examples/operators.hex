a(u).
a(v).
b(u).

c(X,Y) :- a(X), b(Y), X <> Y.
d(X,Y) :- a(X), b(Y), X != Y.

