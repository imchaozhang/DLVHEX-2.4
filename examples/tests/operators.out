{d(v,u), c(v,u), b(u), a(v), a(u)}
